README.md

```diff
@@ -0,0 +1,109 @@
+# BCDC-Dashboards
+This repository is intended to store the codes for the BICAN dashboard, which takes the BCDC-Metadata repository as the input, performs data wrangling and launches the dashboard app script. The current version is a demo version that lives in the local environment and has not been integrated with the other pipelines yet, hence, the code structures are subject to change once the integration with the rest of the existing pipelines (i.e. databases, ontologies) happens.
+
+### Disclaimer: the current dashboard script only demonstrates two use cases of collapsing the categories - 1. collapse specimen types for specimen counts; 2. collapse species for sample counts. There are other categories needing to be collapsed, such as modalities and techniques, which is work in progress - meanwhile, an abstract of what categories to collapse into has been included in the business logics.
+
+
+### Workflow
+1. Input: BCDC-Metadata
+2. Data Wrangling: dashboard_data_wrangling.ipynb
+    Output: dash_count_df.csv
+3. Dashboard script: tast_dash_counts.py (takes dash_count_df.csv as input)
+    Output: Dashboard page with visualizations
+
+### Business logics:
+
+[dashboard_logics.pdf](https://github.com/AllenInstitute/BCDC-Dashboards/files/11033385/dashboard_logics.pdf)
+
+Currently, we are aiming to count the below units:
+1. Number of samples received for each sample type per group - for example, how many brain/cell related samples did we receive per quarter; or for each grant?
+2. Number of donors/subjects per group - for example, how many donors/subjects per quarter/modality/technique?
+3. Number of subspecimens (brains, cells, tissue samples, etc.) processed per quarter/grant/technique/etc.
+
+While counting, the groups and categories will be collapsed into similar terminologies - for example, 'whole cell' and 'cell body' both mean the same thing, and as a result, they will be collapsed together under 'cells' and their counts will be counted towards the same category. 
```

Comment:
I'm not sure that `whole cell` and `cell body` mean the same thing. IIRC, `cell body` is a soma, `whole cell` would include dendrites and axons (e.g.).